core/src/main/kotlin/fr/sncf/osrd/api/api_v2/conflicts/ConflictDetectionResponse.kt

```diff
@@ -9,22 +9,47 @@ import fr.sncf.osrd.utils.json.UnitAdapterFactory
 import java.time.ZonedDateTime
 
 class ConflictDetectionResponse(
+    /**
+     * List of all detected conflicts, if any. Conflicts are grouped between each set of conflicting
```

Comment:
s/grouped between/grouped by/

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/9391/commits/649f0b097ad6f6412d4e5a8a1842faef22ee3e70